recipes/aria2/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH=${PREFIX}/lib/pkgconfig
+
+if [ `uname` == Darwin ]; then
+    # Needed for C++11 support.
+    export MACOSX_DEPLOYMENT_TARGET=10.7
```

Comment:
I don't think this is necessary anymore?
Otherwise you'd need to add a dependency on `__osx=...` in the meta.yaml (check conda-forge knowledge base for that).